apps/nova/src/pages/Tutors.tsx

```diff
@@ -13,7 +14,11 @@ const Tutors: React.FC = () => {
       <PageTitle
         title={intl("tutors.title")}
         fetching={tutors.query.isFetching && !tutors.query.isLoading}
-        className="mb-6"
+        className={cn(
+          "mb-6",
+          { "mb-36": tutors.query.isLoading },
```

Comment:
This is not how we do it. [This how we agreed to do it](https://github.com/litespace-org/litespace/blob/8691a039f6b6586545a489d1f6d9db84e5f480b6/apps/nova/src/components/UpcomingLessons/Content.tsx#L67-L69).

Any **_page-level loading or error_** should have a margin top of `15vh`. This is to position the element based on the user screen.